pupil-api/src/run-jasmine.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
```

Comment:
This whole file (`pupil-api/src/run-jasmine.js`) is a duplicate of `pupil-api/run-jasmine.js`, which is unnecessary since the first one will get executed.

Reply:
good catch, thanks :)